package-lock.json

```diff
@@ -9309,7 +9309,7 @@
       }
```

Comment:
We need to include the updated hashes for `mattermost-redux` in here. Can we do that and revert all other changes?

Reply:
ok

Reply:
fixed in 46ae22d477